components/NavigationBar.tsx

```diff
@@ -0,0 +1,175 @@
+'use client';
+
+import React, { useState } from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+import { usePathname } from 'next/navigation';
+import { useTheme } from 'next-themes';
+
+import ThemeSwitcher from './ThemeSwitcher';
+import { close, open, openWhite, closeWhite } from '../public/assets/index';
+import { logo } from '@/public/assets-png';
```

Comment:
stick to using only one type of import -- I recommend using alias imports as you have in line 11